tools/travis/build.sh

```diff
@@ -21,7 +21,7 @@ $ANSIBLE_CMD apigateway.yml
 
 cd $ROOTDIR
 
-./gradlew distDocker -x :core:swift3Action:distDocker -x :core:pythonAction:distDocker -PdockerImagePrefix=testing
```

Comment:
Inspect this line: https://github.com/openwhisk/openwhisk/pull/2070/files#diff-31a3b6664356184ce7bd3d6d5d6ef4d0R39

This test suite is already running (it's included in `testLean` already).